lib/osf-components/addon/components/activity-log/-components/activity-log-display/component.ts

```diff
@@ -0,0 +1,600 @@
+            pointer_category: this.getPointerCategory(),
+            preprint: this.buildPreprintUrl(),
+            preprint_provider: this.buildPreprintProviderUrl(),
+            preprint_word: this.intl.t('activity-log.defaults.preprint'),
+            preprint_word_plural: this.intl.t('activity-log.defaults.preprint-plural'),
+            source: this.buildSource(),
+            tag: this.buildTagUrl(),
+            template: this.getEmbeddedUrl(),
+            title_new: this.buildTitleNew(),
+            title_original: this.getEmbeddedUrl(),
+            updated_fields: this.buildUpdatedFields(),
+            user: this.buildUserUrl(),
+            value: this.log?.params?.value,
+            version: this.buildVersion(),
+            /*
+            group: null,
+            */
+
+        }) as string;
+
+        return `<span>${translation}</span>`;
+
+    }
+
+    /**
+     * buildAnonymous
+     *
+     * @description Abstracted method to build the anonymous string
+     *
+     * @returns a formatted string
+     */
+    private buildAnonymous(): string {
+        return this.log?.params?.anonymousLink ?
+            this.intl.t('activity-log.defaults.anonymous_an') :
+            this.intl.t('activity-log.defaults.anonymous_a') ;
+    }
+
+    /**
+     * buildCommentLocation
+     *
+     * @description Abstracted method to build the comment location
+     *
+     * @returns a formatted string
+     */
+    private buildCommentLocation(): string {
+        const file = this.log?.params?.file;
+        const wiki = this.log?.params?.wiki;
+        if (file) {
+            return this.intl.t('activity-log.defaults.file_on', {
+                file: this.buildAHrefElement(`/${file.url}`, file.name),
+            });
+        }
+
+        if (wiki) {
+            return this.intl.t('activity-log.defaults.wiki_on', {
+                wiki: this.buildAHrefElement(`/${wiki.url}`, wiki.name),
+            });
+
+        }
+
+        return '';
+    }
+
+
+    /**
+     * buildContributorlist
+     *
+     * @description Abstracted method to build the contributor list
+     *
+     * @returns a formatted string
+     */
+    private buildContributorsList(): string {
+        if (this.log?.params?.contributors) {
+            const contributors = this.log.params.contributors;
+            const maxShown = 3;
+            const contribList: string[] = [];
+
+            // Determine if we are showing the second-to-last contributor (just one more)
+            const isJustOneMore = contributors.length === maxShown + 1;
+
+            // Loop through each contributor
+            for (let i = 0; i < contributors.length; i++) {
+                const contributor = contributors[i];
+                let separator = '';
+
+                // Handle the separator logic for commas or "and"
+                if (i < contributors.length - 1) {
+                    separator = (i === maxShown - 1 && !isJustOneMore) ? ', and ' : ', ';
```

Comment:
a bit of a nit, but this is using the untranslated `and` here. Could use `this.intl.t('general.and')`